DOCS/man/input.rst

```diff
@@ -3573,6 +3573,19 @@ Property list
     and not using raw mode, the underlying content will be given (e.g. strings will be
     printed directly, rather than quoted and JSON-escaped).
 
+    ``user-data/ytdl``
```

Comment:
Nice catch.

Reply:
> Can you change to `user-data/mpv/ytdl` instead? That's the designated path for mpv internal scripts, to avoid messing with 3rd-party uses of `user-data`. 

Interesting, I thought ytdl would be enough to but we can have it under mpv.

Reply:
Per d302fe0de55aeb40e398d1b162f9636eaf07530c, everything mpv-internal should be added under `user-data/mpv` (`user-data/osc` is a grandfathered exception).